spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -133,7 +133,7 @@ interface ErrorReceipt {
 }
 ```
 
-- `sequence` contains the sequence at which the error occurred. Both chains are expected to increment to the next sequence after the upgrade is aborted.
```

Comment:
After we discussed in the engineering call, I mentioned this to @adiraviraj and he agreed to not increment the sequence in `cancelChannelUpgrade`.